### ISSUE TITLE
Midi volume slider default is higher than is possible with gui.

### ISSUE THREAD

**Issue by VerinSenpai** [space-wizards/space-station-14#45780]
### Description

The slider default is .5

Upon updating the value, it drops to a max of .25 which matches the multiplier set here https://github.com/space-wizards/space-station-14/blob/8375450b5a278e4538ae1b6065a8b3dd1cbaf756/Content.Client/Audio/ContentAudioSystem.cs#L26

Personally I think .25 is a touch quiet but regardless the default value shouldn't be unobtainable using the gui sliders.

Audio level demonstration:

https://github.com/user-attachments/assets/a6f16d19-6d9b-488f-9bb6-ee4bf4b833fe

Default value mismatch:

https://github.com/user-attachments/assets/4c5779f0-f127-4a45-abd8-dd1583625336

### Reproduction

Open the console and check `cvar midi.volume`

Note that it is set to .5 (if you haven't changed it yourself at some point)

Open the settings, bump the midi slider and save.

Note the volume is now .25 or less

**Comment by VerinSenpai** [comment]
oh my god the bot is back ;(

**Comment by VerinSenpai** [comment]
I'll note here that the comment above the gain multipliers says this value should be getting multiplied when being set but it instead gets divided and then multiplied in the get scale operation.

**Comment by VerinSenpai** [comment]
whoop misclick.
